src/main/java/io/github/thebusybiscuit/slimefun4/core/multiblocks/MultiBlockMachine.java

```diff
@@ -165,6 +165,29 @@ public void load() {
         }
     }
 
+    /**
+     * This method handles a output {@link ItemStack} from the {@link MultiBlockMachine} which has a crafting delay
+     *
+     * @param outputItem
+     *          A crafted {@link ItemStack} from {@link MultiBlockMachine}
+     * @param dispenser
+     *          Our {@link Dispenser} from {@link MultiBlockMachine}
+     *
+     */
+    protected void handleCraftedItem(ItemStack outputItem, Dispenser dispenser) {
```

Comment:
I would prefer we use `Container` instead of `Dispenser`. I don't see a reason to limit this to ONLY dispensers. This could easily be a chest, trapped chest, furnace even.

Reply:
Okay, I will change it